src/Utils/Helpers.ts

```diff
@@ -233,6 +235,26 @@ export class Helpers {
     }, {});
   }
 
+  public static excludeEntryPointCoerce(args: string[]): RequestExclusion[] {
```

Comment:
This is helper class with common util methods. Why did you move this logic, related to the scan only, this way?

Reply:
Could you explain where I can move this method (and still test it)?